client/src/components/ElementEditor/ElementActions.js

```diff
@@ -93,7 +121,7 @@ class ElementActions extends Component {
       >
         { this.renderEditTabs() }
         { this.renderDivider() }
-        { children }
```

Comment:
I would add a "Context Provider" around `children` here. The context could tell sub component if they are rendered for an inline element or not.